dotnet/src/Planners/Planners.Core/Action/IActionPlanner.cs

```diff
@@ -3,8 +3,9 @@
 using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.SemanticKernel.Diagnostics;
+using Microsoft.SemanticKernel.Planning;
 
-namespace Microsoft.SemanticKernel.Planning.Action;
+namespace Microsoft.SemanticKernel.Planners.Action;
```

Comment:
Think this should be in `namespace Microsoft.SemanticKernel.Planners` and do same for other planner implementations

Reply:
Fixed